File: C99/1000-1099/1020-1029/1021.c
#include <stdio.h>

int main()
{
    int i, j, quant, vez = 0;
    double n;
    double din[12] = {100, 50, 20, 10, 5, 2, 1, 0.5, 0.25, 0.1, 0.05, 0.01};
    scanf("%lf", &n);
    n += 0.001;
    printf("NOTAS:\n");
    for (i = 0; i < 12; i++)
    {
        j = 0;
        quant = 0;
        while (j == 0)
        {
            if (n >= din[i])
            {
                n -= din[i];
                quant++;
            }
            else
            {
                j++;
            }
        }
        if (i <= 5)
        {
            printf("%d nota(s) de R$ %.2lf\n", quant, din[i]);
        }
        else
        {
            if (vez == 0)
            {
                printf("MOEDAS:\n");
                vez++;
            }
            printf("%d moeda(s) de R$ %.2lf\n", quant, din[i]);
        }
    }
    return 0;
}

File: beecrowd/1000-1099/1020-1029/1021.py
entry = input()
money = [100, 50, 20, 10, 5, 2]
coins = [100, 50, 25, 10, 5, 1]
values = entry.split('.')
values = [eval(i) for i in values]
print("NOTAS:")
for i in money:
    print(f"{int(values[0] / i)} nota(s) de R$ {i:.2f}")
    values[0] %= i
if values[0] == 1:
    values[1] += 100
print("MOEDAS:")
for i in coins:
    print(f"{int(values[1] / i)} moeda(s) de R$ {i / 100:.2f}")
    values[1] %= i